format

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+find src -type f | egrep -v '(lib\/.*\/|linux|archives)' | egrep '\.[ch]$' | xargs clang-format -i
```

Comment:
I agree, but I think that's a next step. I'd like to get this merged before doing all that infrastructure work.